scipy/optimize/_qap.py

```diff
@@ -636,55 +606,44 @@ def _quadratic_assignment_2opt(A, B, maximize=False, partial_match=None,
     Options
     -------
 
-    maximize : bool (default = False)
+    maximize : bool (default: False)
         Setting `maximize` to ``True`` solves the Graph Matching Problem (GMP)
-        rather than the Quadratic Assingnment Problem (QAP).
-
-    rng : {None, int, `~np.random.RandomState`, `~np.random.Generator`}
-        This parameter defines the object to use for drawing random
-        variates.
-        If `rng` is ``None`` the `~np.random.RandomState` singleton is
-        used.
-        If `rng` is an int, a new ``RandomState`` instance is used,
-        seeded with `rng`.
-        If `rng` is already a ``RandomState`` or ``Generator``
-        instance, then that object is used.
-        Default is None.
-
-    partial_match : 2d-array of integers, optional, (default = None)
-        Allows the user to fix part of the matching between the two
-        matrices. In the literature, a partial match is also known as a
-        "seed".
-
-        Each row of `partial_match` specifies the indices of a pair of
-        corresponding nodes, that is, node ``partial_match[i, 0]`` of `A` is
-        matched to node ``partial_match[i, 1]`` of `B`. Accordingly,
-        ``partial_match`` is an array of size ``(m , 2)``, where ``m`` is
-        not greater than the number of nodes.
-
-    partial_guess : 2d-array of integers, optional, (default = None)
-        Allows the user to provide a guess for the matching between the two
-        matrices. Unlike `partial_match`, `partial_guess` does not fix the
-        indices; they are still free to be optimized.
-
-        Each row of `partial_guess` specifies the indices of a pair of
-        corresponding nodes, that is, node ``partial_guess[i, 0]`` of `A` is
-        matched to node ``partial_guess[i, 1]`` of `B`. Accordingly,
-        ``partial_guess`` is an array of size ``(m , 2)``, where ``m`` is
-        less than or equal to the number of nodes.
+        rather than the Quadratic Assignment Problem (QAP).
+
+    rng : int, `RandomState`, `Generator` or None, optional (default: None)
+        Accepts an integer as a seed for the random generator or a
+        ``RandomState`` or ``Generator`` object. If None (default), uses
+        global `numpy.random` random state.
+
+    partial_match : 2-D array of integers, optional (default: None)
+        Fixes part of the matching. Also known as a "seed" [2]_.
```

Comment:
['scipy.optimize.quadratic_assignment', ':impl: scipy.optimize._qap._quadratic_assignment_2opt', ':method: 2opt']:71: WARNING: Unknown target name: "2".
/home/circleci/repo/doc/source/optimize.qap-faq.rst:92: WARNING: Inline interpreted text or phrase reference start-string without end-string.

Need to add [2] to the references of this function.

Reply:
Thanks for debugging that. I was getting confused with `[2]` and `2opt` - thought it was complaining about the latter.